src/averageAges.js

```diff
@@ -37,7 +43,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const reducer = (arrs) => arrs.reduce((elem, total) => elem + total, 0);
```

Comment:
you can move it outside of the function if it's a helper function or just run the code here